### ISSUE TITLE
Document what happens when tags are deleted and recreated

### ISSUE THREAD

**Issue by aofarrel** [dockstore/dockstore#4721]
_This is a WIP..._

# Deleting a tag
1. [Make a commit on Fairyland](https://github.com/aofarrel/fairyland/commit/23ac8bd7fddc4ea68fda30861755c283bd66f083), which is a Dockstore.yml mediated collection of "workflows"
2. Make a release on GitHub called "semantic versioning is fake"
3. Import into Terra and select semantic-versioning-is-fake to ensure it is new default, see figure A
4. [Make another commit on Fairyland](https://github.com/aofarrel/fairyland/commit/bb0a83d10b60fefdfa8bea68ba7cc8a95e27a113)
5. Delete the GitHub release, but do not undo any commit history

Observation A: [Dockstore's URL for the deleted tag](https://dockstore.org/workflows/github.com/aofarrel/fairyland/wdl-pruning:semantic-versioning-is-fake?tab=versions) now redirects [to main](https://dockstore.org/workflows/github.com/aofarrel/fairyland/wdl-pruning:main?tab=versions)
Observation B: Terra throws an error when selecting the workflow, see figs B and C. Contrary to what the error message says, you might be able to select main from the dropdown menu, but let's not do that right now, let's keep the "default" to the nonexisting tag.
All of this is more or less what you would expect and, in my eyes, acceptable behavior.

### Figure A
<img width="1371" alt="Screen Shot 2022-02-10 at 2 04 34 PM" src="https://user-images.githubusercontent.com/27784612/153508226-403915d5-90a1-4279-a90c-495864f18b44.png">

### Figure B
<img width="1360" alt="Screen Shot 2022-02-10 at 2 39 23 PM" src="https://user-images.githubusercontent.com/27784612/153508900-513fe6fd-d9fa-4345-a754-a572924294e2.png">

### Figure C
<img width="791" alt="Screen Shot 2022-02-10 at 2 39 33 PM" src="https://user-images.githubusercontent.com/27784612/153508937-4862c35f-4b38-493c-b6ff-3794b4a9c7f7.png">

# Recreating a tag

**Comment by denis-yuen** [comment]
Agreed, not really an issue when snapshots https://docs.dockstore.org/en/stable/advanced-topics/snapshot-and-doi.html are not in play. That said, that experience needs also to be better, see also #4559 or #3211
